### ISSUE TITLE
Icons that need a redesign

### ISSUE THREAD

**Issue by daniruiz** [daniruiz/flat-remix#160]
Write here the icons you find outdated or incorrect

It helps me a lot if you can write the exact name of the icon. Even more if you can provide an alternative icon (a screenshot of it is enough) from a different icon theme

**Comment by maheer-kolcha** [comment]
system monitor

**Comment by daniruiz** [comment]
@maheer-kolcha what happens to system-monitor?

**Comment by maheer-kolcha** [comment]
it is not Immediately recognisable as system moniter

**Comment by daniruiz** [comment]
Could you provide a screenshot of the icon and the name of the application you're referring to?

**Comment by daniruiz** [comment]
Sorry, but I think it's perfectly recognisable
Flat-Remix monitor icon
![image](https://user-images.githubusercontent.com/12565871/96226431-e5cd2200-0f92-11eb-9c8e-3e68891e6aa1.png)
Default icon
![image](https://user-images.githubusercontent.com/12565871/96226452-f087b700-0f92-11eb-920a-91770957994c.png)

**Comment by maheer-kolcha** [comment]
yes i was talking about this icon, it is very similar to image/imageviewer
when it is small like in cinnamon panel.

**Comment by Jocix123** [comment]
Master PDF Editor. :)

**Comment by ppmathis** [comment]
Riot (instant messenger for Matrix) has been renamed to Element and also switched their icon while doing so, which has not yet been reflected in this icon pack. I've installed Element using Flatpak and it seems to still use `im.riot.Riot.svg`, causing the old display from this icon pack to be displayed.

Element/Riot in Flat Remix:
<img src="https://zig.pw/0829823a7c82b4a80fac41deca8e703a093fa88b.svg" width="64" />

Element default icon:
<img src="https://zig.pw/e746bc2ac8a898b4922d72be435b6c805e50a5d8.png" width="64" />

**Comment by ProjektRoot** [comment]
Hi :)
distributor-logo-manjaro.svg and manjaro-welcome.svg icons needs update (/.../apps/scalable/). I think you can use [maia.svg](https://gitlab.manjaro.org/artwork/icon-themes/manjaro-icons/-/blob/master/maia/maia.svg) or something else from [branding directory](https://gitlab.manjaro.org/artwork/branding/logo) or [manjaro-icons](https://gitlab.manjaro.org/artwork/icon-themes/manjaro-icons).
Great job, kudos to you :)

**Comment by daniruiz** [comment]
* Virtualbox
* Terminal icons

**Comment by daniruiz** [comment]
* All display icons

**Comment by daniruiz** [comment]
* Characters
* Keyboard icons

**Comment by daniruiz** [comment]
User icon
![image](https://user-images.githubusercontent.com/12565871/118842628-ad474b00-b8c9-11eb-9bc8-fca149847b73.png)

**Comment by Trikenstein** [comment]
Virtual Machine Manager (apps/virt-manager.svg)
Original: !{virt-manager](https://avatars.githubusercontent.com/u/12552438?s=200&v=4)

**Comment by Trikenstein** [comment]
apps/gnome-tweaks.svg

I think a better icons should have the gnome logo in it. Example: https://icon-icons.com/icon/gnome-tweak-tool/103048
![gnometweaktool_103048](https://user-images.githubusercontent.com/72353094/119578706-06f8bb00-bd8b-11eb-9a30-f2db5de2a509.png)

**Comment by rcmz** [comment]
apps/scalable/deezer.svg
apps/scalable/nuvolaplayer3_deezer.svg
apps/scalable/eu.tiliado.NuvolaAppDeezer.svg

old: https://svgur.com/s/Xb9
proposed new: https://svgur.com/s/Xbb

**Comment by Trikenstein** [comment]
@rcmz FYI you can scale down the original size, using HTML syntax:

```
<img src="https://svgur.com/s/Xb9" alt="Orig deezer" width="100"/>
```

<img src="https://svgur.com/s/Xb9" alt="Orig deezer" width="100"/>

**Comment by rcmz** [comment]
@Trikenstein thanks !

**Comment by daniruiz** [comment]
rclone
logs
jabref
stremio
glabels
imagej1
qnapi
moodbar
liferea
deluge
labplot2
zenmap
scintilla
systemsettings

**Comment by Yisus7u7** [comment]
![Screenshot_20210730-201519](https://user-images.githubusercontent.com/64093255/127724212-0b8e95f0-91c4-4f62-b0ae-fa33e20ff376.png)

An icon on the termux app wouldn't be bad at all

**Comment by poikilos** [comment]
- Distro: Devuan
- Desktop: MATE
- moved from: https://github.com/daniruiz/skeuos-gtk/issues/31

There is an up and down arrow for open and save. The buttons aren't intuitive. I understand the symbolism of down going down to the drive or something, but in a world of web applications the buttons are counter-intuitive (and I recently almost overwrote an important file):
- Open being an up arrow is unexpected because an *up arrow*  can mean upload, but you are "downloading" in a symbolic way.
- Save and Save All being a down arrow and down arrow with "A" over it are unexpected because a *down arrow*  can mean download, but you are "uploading" in a symbolic way.

I tried:
- to read documentation and unofficial documentation of creating GTK themes
- search for "open" "save" "load" and "arrow.up" and "arrow.down" in the css files of the theme, but I can't figure out how to change the icons.
- visually look at every icon included in the theme

The Skeuos maintainer says the metaphor is that loading it into the program is on a "higher level" but like I said, I understand the metaphor, it just isn't common. If you have to explain it or it is ambiguous, that is a problem that could cause data loss and almost did for me.

Here are the up and down arrows in Geany:
![image](https://user-images.githubusercontent.com/7557867/144612465-c882efa9-86db-4c86-b47e-5e8569d5add8.png)

Notice there are 3 icons involved, and I tried hard and couldn't find the names.

**Comment by poikilos** [comment]
@daniruiz :confused:   Why do you offer a thumbs down without even:
- the logic behind the nonsensical icons
- the consideration of telling me where the icons can be changed if I fork the theme

**Comment by daniruiz** [comment]
@poikilos because I had already discussed with you why it made no sense. A down arrow references save to disk, an up arrow means load from disk. It's always been that way.
Anyway, the got changed in the latest update to a more clear symbol so it might be fixed for you now :wink: 
Hope that helps

**Comment by bungadrum** [comment]
Video Downloader, Yoga Image optimizer, RecApp 
![Screenshot_2022-05-26_15-16-52](https://user-images.githubusercontent.com/75974100/170495664-88325934-da7c-4d5c-8667-ae5fc330e91e.png)
![Screenshot_2022-05-26_15-18-27](https://user-images.githubusercontent.com/75974100/170495666-2699bbd8-7e39-4fd4-a74d-4b92f543872e.png)
![Screenshot_2022-05-26_15-18-43](https://user-images.githubusercontent.com/75974100/170495670-ee16d1b0-4d8f-43ff-8edd-01fbbe264f2c.png)

**Comment by snapfast** [comment]
- App Name: Sublime text
- Icon Name: sublime-text
- Distribution: LMDE

The icon is old, update to the latest version.

![image](https://user-images.githubusercontent.com/5176514/183502445-d07ca513-f7c4-44ca-bc15-11abb9a84d55.png)

**Comment by snapfast** [comment]
- App Name: Terminal
- Icon Name: org.gnome.Terminal
- Distribution: LMDE

The icon is old, update to the latest version.

![image](https://user-images.githubusercontent.com/5176514/183503521-ae6b8735-25da-4a05-bee0-7a85bb1d7e2d.png)

**Comment by andras168** [comment]
It would be nice to get different icon for Krusader and Kitty terminal emulator. Kitty has the same icon as the default terminal (konsole in my case). Steam icon is ugly as well.

**Comment by 3hr80FEZwlir** [comment]
Some time ago the LibreOffice updated it's icons.

Here are new icons:
https://wiki.documentfoundation.org/images/6/68/NewAppMimeIcons.png

Thank you for - IMO - the best icon pack for Linux!

**Comment by skyw4ll** [comment]
> * App Name: Sublime text
> 
>     * Icon Name: sublime-text
> 
>     * Distribution: LMDE
> 
> 
> The icon is old, update to the latest version.
> 
> ![image](https://user-images.githubusercontent.com/5176514/183502445-d07ca513-f7c4-44ca-bc15-11abb9a84d55.png)

+1
